tests/tracing/test_fluent.py

```diff
@@ -148,11 +150,14 @@ def square(self, t):
     assert trace_info.status == TraceStatus.OK
     assert trace_info.request_metadata[TraceMetadataKey.INPUTS] == '{"x": 2, "y": 5}'
     assert trace_info.request_metadata[TraceMetadataKey.OUTPUTS] == "64"
-    assert trace_info.tags == {
+
+    expected_tags = {
         "mlflow.traceName": "predict",
         "mlflow.artifactLocation": "test",
         "mlflow.user": "bob",
     }
+    expected_tags.update(registry.resolve_tags())
+    assert trace_info.tags == expected_tags
```

Comment:
In the test can we verify the concrete tag items added by `resolve_tags` ?

Reply:
We can but it may cause the tests to fail locally because the hardcoded items can change from system to system.

I could mock all the env vars as a workaround but was worried about it being verbose